### ISSUE TITLE
Multiparts and multiple Instances

### ISSUE THREAD

**Issue by kiegel** [lcnetdev/marc2bibframe#179]
We have discussed generating multiple Instances for multipart monographs on another issue (sorry I can't locate it now), but I want to show an example where it does not make sense to us.

We have a 12 volume set:

020	__ |a 9787549554188 (v. 1)
020	__ |a 7549554188 (v. 1)
...
020	__ |a 9787549518395 (v. 12)
020	__ |a 7549518394 (v. 12)
...
100	1_ |a Yu, Yingshi, |e author.
240	10 |a Works. |k Selections. |f 2014
245	10 |a Yu Yingshi wen ji / |c Yu Yingshi zhu ; Shen Zhijia bian.
250	__ |3 Volume 1-10: |a Di 2 ban.
250	__ |3 Volume 11-12: |a Di 1 ban.
264	_1 |a Guilin Shi : |b Guangxi shi fan da xue chu ban she, |c 2014-
300	__ |a volumes ; |c 22 cm
...
505	1_ |a Di 1 juan. Shi xue, shi jia yu shi dai -- Di 2 juan. Zhongguo si xiang chuan tong ji qi xian dai bian qian -- Di 3 juan. Ru jia lun li yu shang ren jing shen -- Di 4 juan. Zhongguo zhi shi ren zhi shi de kao cha -- Di 5 juan. Xian dai xue ren yu xue shu -- Di 6 juan. Min zhu zhi du yu xian dai wen ming -- Di 7-8 juan. Wen hua ping lun yu Zhongguo qing huai (shang, xia). -- Di 9 juan. Li shi ren wu kao bian -- Di 10 juan. Song Ming li xue yu zheng zhi wen hua -- Di 11 juan. Lun xue hui you -- Di 12 juan. Guo xue yu Zhongguo ren wen
...

It generates 12 Instances (instance21 through instance32).  Each of them is identical except for ISBNs and an ISBN qualifier as part of bf:title.

Here is the first one:
~~~xml
<http://example.org/99161784508601452instance21> a bf:Instance,
        bf:Monograph ;
    bf:carrierCategory <http://id.loc.gov/vocabulary/carriers/nc> ;
    bf:contentsNote "Di 1 juan. Shi xue, shi jia yu shi dai -- Di 2 juan. Zhongguo si xiang chuan tong ji qi xian dai bian qian -- Di 3 juan. Ru jia lun li yu shang ren jing shen -- Di 4 juan. Zhongguo zhi shi ren zhi shi de kao cha -- Di 5 juan. Xian dai xue ren yu xue shu -- Di 6 juan. Min zhu zhi du yu xian dai wen ming -- Di 7-8 juan. Wen hua ping lun yu Zhongguo qing huai (shang, xia). -- Di 9 juan. Li shi ren wu kao bian -- Di 10 juan. Song Ming li xue yu zheng zhi wen hua -- Di 11 juan. Lun xue hui you -- Di 12 juan. Guo xue yu Zhongguo ren wen" ;
    bf:dimensions "22 cm" ;
    bf:edition "Di 1 ban.",
        "Di 2 ban." ;
    bf:instanceOf <http://example.org/99161784508601452> ;
    bf:instanceTitle <http://example.org/99161784508601452title27> ;
    bf:isbn10 <http://isbn.example.org/7549554188> ;
    bf:isbn13 <http://isbn.example.org/9787549554188> ;
    bf:mediaCategory <http://id.loc.gov/vocabulary/mediaTypes/n> ;
    bf:modeOfIssuance "single unit" ;
    bf:providerStatement "Guilin Shi : Guangxi shi fan da xue chu ban she, 2014-" ;
    bf:publication [ a bf:Provider ;
            bf:providerDate "2014-" ;
            bf:providerName [ a bf:Organization ;
                    bf:label "Guangxi shi fan da xue chu ban she" ;
             bf:providerPlace [ a bf:Place ;
                    bf:label "Guilin Shi "  ] ] ;
    bf:responsibilityStatement "Yu Yingshi zhu ; Shen Zhijia bian." ;
    bf:supplementaryContentNote "Includes bibliographical references." ;
    bf:systemNumber <http://www.worldcat.org/oclc/891156210> ;
    bf:title "Yu Yingshi wen (v. 1)" ;
    bf:titleStatement "Yu Yingshi wen ji" .

<http://example.org/99161784508601452title27> a bf:Title ;
    bf:titleValue "Yu Yingshi wen ji" .
```

When we look at this instance, we see a description of the *set* rather than volume 1.  The contents note is the contents of the set, not volume 1.  The edition statement is for the set.  This instance, which is supposed to be volume 1, would be part of the work, not an instance of it.  The instance title is the title of the set, not volume 1.  The provider statement is the publication information of the set, not just volume 1 (i.e. not a single year date but a range).  The responsibility statement is for the set.  The OCLC number is the number of the set.  titleStatement is for the set.

In other words, as a description of volume 1, this instance fails.  The other eleven instances also fail as descriptions of their respective volumes.

To us, this example seriously calls into question the approach of creating item (volume) level descriptions from MARC records.  MARC cataloging is at the manifestation level and should be converted that way.

(OCLC # 891156210)

**Comment by ntra00** [comment]
I updated the transformation so that you can toggle the option to create Instances for each isbn pair (10&13). If you want to do this, set the status to "on".

Default status is "off".

<rule status="off" id="1" label="isbn" category="instance-splitting">New instances on secondary unique ISBNs</rule>
